client/src/general_utils.ts

```diff
@@ -3,6 +3,8 @@ import JSURL from "jsurl";
 import _ from "lodash";
 import marked from "marked";
 
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
`any` in this file for `this` keyword. Not sure how to make decorators type safe